target/linux/ath79/dts/qca9531_glinet_gl-ar300m-nor.dts

```diff
@@ -1,11 +1,15 @@
-/dts-v1/;
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
You need an ACK from all previous authors of this file to do a license changing.

Reply:
Will remove next time I tackle this